src/components/Table/data/groupBy.ts

```diff
@@ -20,16 +20,37 @@ const data: DataType[] = [
         id: 'id0000011',
         name: 'Order 11',
         createdDate: '2020-01-01',
+        subRows: [
+            {
+                id: 'id0000111',
+                name: 'Order 111',
+                createdDate: '2020-01-01',
+            },
+        ],
     },
     {
         id: 'id0000012',
         name: 'Order 12',
         createdDate: '2020-01-01',
+        subRows: [
+            {
+                id: 'id0000211',
+                name: 'Order 211',
+                createdDate: '2020-01-01',
+            },
+        ],
     },
     {
         id: 'id0000013',
         name: 'Order 12',
         createdDate: '2020-01-02',
+        subRows: [
```

Comment:
Does the nested item prop has to call *subRows*? Is this configurable?

And BTW, the build is also failed here: 

/codebuild/output/src715/src/github.com/aws/aws-northstar/src/components/Table/data/groupBy.ts
--
428 | [tsl] ERROR in /codebuild/output/src715/src/github.com/aws/aws-northstar/src/components/Table/data/groupBy.ts(23,9)
429 | TS2322: Type '{ id: string; name: string; createdDate: string; subRows: { id: string; name: string; createdDate: string; }[]; }' is not assignable to type 'DataType'.
430 | Object literal may only specify known properties, and 'subRows' does not exist in type 'DataType'.
431 | /codebuild/output/src715/src/github.com/aws/aws-northstar/src/components/Table/data/groupBy.ts

Reply:
@jessieweiyi `subRows` is the property for external expand logic of [react-tables](https://react-table.tanstack.com/docs/api/useExpanded#useexpanded)

Reply:
Thanks. Might be good to include the 

```getSubRows: Function(row, relativeIndex) => Rows[]
Optional
See the useTable hook for more details```

So that users do not have to change the data structure to accommodate the feature.